Gemfile

```diff
@@ -61,6 +61,8 @@ gem 'shell-spinner', '~> 1.0', '>= 1.0.4'
 gem 'ruby-progressbar'
 gem 'geckoboard-ruby'
 gem 'posix-spawn', '~> 0.3.13'
+# gem 'laa-fee-calculator-client', :git => 'git@github.com:ministryofjustice/laa-fee-calculator-client.git', :branch => 'master'
```

Comment:
**[not a blocker]** ✂️since we're now using the line below that used tagged versions.